packages/frontend/src/common/components/NavigationItem.tsx

```diff
@@ -0,0 +1,24 @@
+'use client'
```

Comment:
I tried few different approaches but haven't found working. Let's leave and I'll fix it in next PR.

Reply:
As discussed, I think it is fine / necessary to use in this component.